co-log-polysemy/co-log-polysemy.cabal

```diff
@@ -82,7 +82,7 @@ library
                            Colog.Polysemy.Effect
 
   build-depends:       co-log-core ^>= 0.2.0.0
-                     , polysemy ^>= 1.0
+                     , polysemy (>= 1.0 && < 1.2)
```

Comment:
Are these `()` parenthesis required? Cabal should be able to handle bounds without them.